.binder/environment.yml

```diff
@@ -0,0 +1,21 @@
+channels:
+  - conda-forge
+dependencies:
+  - ase
+  - coveralls
+  - coverage
+  - codacy-coverage
+  - damask
+  - matplotlib
+  - fenics
+  - mshr
+  - numpy
+  - pyiron
+  - pyiron-data
```

Comment:
It is for the binder environment.

Reply:
Can't we load the `environment-notebooks.yml` instead? Since the `environment.yml` is the file we normally use to tell everyone what packages we expect.

Reply:
Sorry, I completely missed the path of this `environment.yml` file, my bad.

Reply:
We can use for our github workflows, this environment file (from `.binder/`) but I don't know about the other way around.

Reply:
I think it is fine like it is right now! I just did not realize this was in the `binder` directory...